package.json

```
@@ -2,11 +2,20 @@
   "name": "root",
   "private": true,
   "devDependencies": {
-    "lerna": "^3.15.0"
+    "husky": "^4.2.3",
+    "lerna": "^3.15.0",
+    "prettier": "^2.0.4",
+    "pretty-quick": "^2.0.1"
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "test": "lerna run test",
-    "lint": "lerna run lint"
+    "lint": "lerna run lint",
+    "format": "pretty-quick"
```

pretty-quick only runs prettier on changed files - I'd prefer to run the format command to run on *all* files.

```suggestion
    "format": "prettier './**/*' --write"
```

Any files that prettier doesn't have a parser for should be added to a `.prettierignore` file

---

I didn't want to be overly aggressive but this sounds good to me.